app/actions/local/post.ts

```diff
@@ -78,7 +79,7 @@ export const sendEphemeralPost = async (serverUrl: string, message: string, chan
             user_id: authorId,
             channel_id: channeId,
             message,
-            type: Post.POST_TYPES.EPHEMERAL_ADD_TO_CHANNEL as PostType,
+            type: Post.POST_TYPES.EPHEMERAL as PostType,
```

Comment:
I guess this changed as the it was originally meant to be `EPHEMERAL` instead of `EPHEMERAL_ADD_TO_CHANNEL` ?